src/data-table/column-numerical.js

```diff
@@ -428,7 +428,13 @@ function NumericalFilter(props) {
 
 const NumericalCell = React.forwardRef<_, HTMLDivElement>((props, ref) => {
   const [css, theme] = useStyletron();
-
+  const formattedValue =
```

Comment:
This change can be encapsulated in the `format` function on L63 with an early return if `props.format` is a function